packages/protocol/contracts/rrp/authorizers/Api3RequesterRrpAuthorizer.sol

```diff
@@ -29,6 +29,22 @@ contract Api3RequesterRrpAuthorizer is
         override(MetaAdminnable, RankedAdminnable, IRankedAdminnable)
         returns (uint256)
     {
-        return MetaAdminnable.getRank(adminnedId, admin);
+        return MetaAdminnable.getRank(bytes32(0), admin);
```

Comment:
The docstring for this function needs to change

Reply:
updated